addons/knowledge/models/knowledge_article_favorite.py

```diff
@@ -28,50 +28,13 @@ def create(self, vals_list):
         a correct ordering as much as possible. Some sequence could be given in create values, that could lead to
         duplicated sequence per user_id. That is not an issue as they will be resequenced the next time the user reorder
         their favorites. """
-        # TODO DBE: Write/change tests to test the create multi with given sequences (that lead to duplicates) + set_sequence that should reorder eveything properly.
-        current_user_id = self.env.uid
-        favorite_without_sequences = [vals for vals in vals_list if 'sequence' not in vals]
-        user_ids = list(set(vals.get('user_id', self.env.context.get('default_user_id')) or current_user_id
-                            for vals in favorite_without_sequences))
-        max_sequence_per_user = self._get_users_max_sequence(user_ids)
+        max_sequence = self.env['knowledge.article.favorite'].search([], order='sequence desc', limit=1).sequence or 0
```

Comment:
would rename it new_sequence
and do search(...).sequence + 1 or 0
to start the sequencing at 0